vk_video_decoder/demos/vk-video-dec/Main.cpp

```diff
@@ -32,6 +32,17 @@ int main(int argc, const char **argv) {
     ProgramConfig programConfig(argv[0]);
     programConfig.ParseArgs(argc, argv);
 
+    // In the regular application usecase the CRC output variables are allocated here and also output as part of main.
+    // In the library case it is up to the caller of the library to allocate the values and initialize them.
+    std::vector<uint32_t> crcAllocation;
```

Comment:
I don't think we need a vector to store the CRC values per frame.
Since you have an output CRC file, I would do the following:
Skip the first 16 bytes:
8 bytes for the size of the file and 8 bytes for the global CRS.
Then for each frame, please write sequentially to the file two 16-bit words:
One for the offset of the output and a second one for the CRC. 
At the end of the stream, just move the file pointer back to the beginning and write the total file size and the global CRC.

Reply:
This doesn't store the per frame values. It's the set of CRC entries, so you can decide to have multiple CRCs generated with different seeds.

Reply:
Oho, I see - I misunderstood the change. - sorry!
But I still would like to see the CRC output file with the above format described, please.

Reply:
Sure, I was planning to do it in a followup change but I guess now is now.